.github/workflows/authors.yml

```diff
@@ -0,0 +1,31 @@
+name: "authors update"
+on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: "5 0 * * 0"
+
+  workflow_dispatch:
+
+jobs:
+  authors_update:
+    if: github.repository == 'sequelize/sequelize'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "0" # This is required to actually get all the authors
+          persist-credentials: false
+      - run: "dev/update-authors.js" # Run the AUTHORS tool
+      - uses: gr2m/create-or-update-pull-request-action@v1 # Create a PR or update the Action's existing PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_USER_TOKEN }}
+        with:
+          author: Node.js GitHub Bot <github-bot@iojs.org>
```

Comment:
Woops, did not notice this one before. I'll check if we have a GH_USER_TOKEN and if so which user is associated with that

Reply:
There is none currently, so we need to make a decision which account we want to use. Haven't encountered a sequelize bot account before, but might be nice to make for this and for the website deployments.
 @fncolon would you be okay if we wait a few days before merging this so @sdepold can look into this when he feels better again and has the time?

Reply:
It could use the GITHUB_TOKEN secret instead https://docs.github.com/en/actions/security-guides/automatic-token-authentication
It's a token for a bot managed by github. I've used it in the past to automatically comment coverage
![image](https://user-images.githubusercontent.com/1280915/149629055-133baa2e-f4f9-4ab2-839b-61ad98d617ff.png)

Reply:
That would be even better!

Reply:
👍👍 

alternatively there is my personal GH_TOKEN

Reply:
Woww that's so many option, but i implement the `GITHUB_TOKEN` thanks for the team work!